packages/amplify-e2e-core/src/utils/index.ts

```diff
@@ -30,7 +30,8 @@ export * from './git-operations';
 /**
  * Whether the current environment is CircleCI or not
  */
-export const isCI = (): boolean => process.env.CI && process.env.CIRCLECI && JSON.parse(process.env.CI) && JSON.parse(process.env.CIRCLECI);
+
+export const isCI = (): boolean => JSON.parse(process.env.CI || 'false') && JSON.parse(process.env.CIRCLECI || 'false');
```

Comment:
Is this change still needed? Looks like `isCI` was updated to handle the cases where these variables are undefined.

Reply:
The previous statement returns undefined when `process.env.CI` and `process.env.CIRCLECI` are undefined. The above statement will always return a boolean, which is the defined return type. Typescript compiler doesn't seem to mind. I'm partial to returning the correct type for better definition of the function, but I don't feel too strongly since the compiler passes.